app/presenters/api/v1/research_output_presenter.rb

```diff
@@ -62,7 +62,7 @@ def fetch_q_and_a(themes:)
         ret = themes.map do |theme|
           qs = @plan.questions.select { |q| q.themes.collect(&:title).include?(theme) }
           descr = qs.map do |q|
-            a = @plan.answers.select { |ans| ans.question_id = q.id }.first
+            a = @plan.answers.select { |answer| answer.question_id = q.id }.first
```

Comment:
ping on this one -- is this legit change or should I keep `ans` here? I do not know ruby